src/js/options.js

```diff
@@ -316,6 +318,23 @@ function updateSocialWidgetReplacement() {
   });
 }
 
+/**
+ * Update DNT checkbox clicked
+ */
+function updateDNTCheckboxClicked() {
+  const enabled = $("#enable_dnt_checkbox").prop("checked");
+
+  chrome.runtime.sendMessage({
+    type: "updateSettings",
+    data: {
+      DNTEnabled: enabled
+    }
+  });
+
+  $("#check_dnt_policy_checkbox").prop("checked", enabled).prop("disabled", enabled);
```

Comment:
When the options page first loads, "Check if sites comply with EFF's Do Not Track policy" is not grayed out, but toggling the send DNT signal checkbox grays it out. This is inconsistent.

If we are going to force-enable DNT checking when DNT sending is enabled, we should it make it visually clear that DNT checking is a sub-option of DNT sending.

Reply:
Thanks, I've fixed this inconsistency.  The grey-out state of the DNT checking is persisted now on initial load.  And disabling sending the DNT header automatically disables the DNT checking.